plugins/modules/ec2_vol_info.py

```diff
@@ -59,14 +59,15 @@
     contains:
         attachment_set:
             description: Information about the volume attachments.
```

Comment:
```suggestion
            description:
            - Information about the volume attachments.
            - This was changed in version 2.0.0 from a dictionary to a list of dictionaries.
```

Reply:
@tremble Ah, missed that. I looked at the boto3 documentation and it is returned as a list of dicts, so I thought it was only a mismatch. Got it know, thanks.